xtask/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "xtask"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1.0.51"
+clap = { version = "3.0.0-rc.7", features = ["derive"] }
```

Comment:
Does this add `clap` to pyo3's dependency tree? If so can we avoid that?

Reply:
I don't believe it does, no. This is an excerpt of `Cargo.lock` from a test project built against this branch:

```
[[package]]
name = "pyo3"
version = "0.15.1"
dependencies = [
 "cfg-if",
 "indoc",
 "libc",
 "num-bigint",
 "parking_lot",
 "paste",
 "pyo3-build-config",
 "pyo3-macros",
 "unindent",
]
```

So `clap` hasn't come through as a dependency here.